tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -243,6 +244,12 @@ def test_nodereplacement_proactive_with_io_running(
         if config.ENV_DATA.get("encryption_at_rest"):
             osd_encryption_verification()
 
+        ocs_version_semantic = version.get_semantic_ocs_version_from_config()
+        if ocs_version_semantic >= version.VERSION_4_13:
+            assert (
+                verify_storagecluster_nodetopology
+            ), "Storagecluster node topology is having an entry of non ocs node(s) - Not expected"
```

Comment:
Looks good. But I don't think we need the condition `if ocs_version_semantic >= version.VERSION_4_13:`, because we anyway use different branches for every release. We just need to backport the changes to the relevant branches(which is 4.13 for this PR).
So, instead, we will have only this code: 

> assert (
                verify_storagecluster_nodetopology
            ), "Storagecluster node topology is having an entry of non ocs node(s) - Not expected"

Same thing for the other occurrences.

Reply:
done